### PR TITLE
downgrade nextra

### PR DESCRIPTION
**Pull request by falcowinkler**
## 🧢 Checklist

- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged

## 📌 Resources

- [Security guidelines](https://github.com/nextauthjs/.github/blob/main/SECURITY.md)
- [Contributing guidelines](https://github.com/nextauthjs/.github/blob/main/CONTRIBUTING.md)
- [Code of conduct](https://github.com/nextauthjs/.github/blob/main/CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

**Comment by ndom91** [comment]
Why do we need to downgrade this?

**Comment by falcowinkler** [comment]
@ndom91 this fixes https://authjs.dev/concepts/oauth hanging / crashing. i tried to find a solution as well without having to downgrade with no success. safari comlains about a syntax error even though the teplate renders fine on the same mermaid version using[ live editor ](https://github.com/mermaid-js/mermaid-live-editor)🤷

**Comment by falcowinkler** [comment]
it must be something specific in the diagram code though that confuses newer version of mermaid / nextra. because other mermaid diagrams run just fine.
Maybe we can convert this specific one to a png to avoid having to downgrade.

**Comment by falcowinkler** [comment]
https://authjs.dev/concepts/database-models works without a problem for example

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -5001,13 +4934,13 @@ packages:
   '@tediousjs/connection-string@0.3.0':
     resolution: {integrity: sha512-d/keJiNKfpHo+GmSB8QcsAwBx8h+V1UbdozA5TD+eSLXprNY53JAYub47J9evsSKWDdNG5uVj0FiMozLKuzowQ==}
 
-  '@theguild/remark-mermaid@0.1.2':
-    resolution: {integrity: sha512-CB1n996M3JqR/9yAOchdOtfmtvDASCUlg/M4wJ/dmZmEpLNlBko8V31Nszo5Uz9+xaO6NrYcjEXA9i/PxzI1xQ==}
```
Comment (87xie) [inline pnpm-lock.yaml:5005]: Hi everyone, this error seems related to `@theguild/remark@0.1.2`, and updating the lockfile to install `@theguild/remark@0.1.3` might resolve the issue. You can find the release [here](https://github.com/the-guild-org/docs/releases/tag/release-1727376017280).

**Pull request merged**